### PR TITLE
added ffmpeg and supervisor

### PR DESCRIPTION
**Pull request by mbehrsing**

### CODE REVIEW

File: 7.2/Dockerfile [hunk 1]
```diff
@@ -84,3 +84,11 @@ WORKDIR /usr/src/app
 RUN apt-get install -y libmagickwand-dev \
     && pecl install imagick \
     && docker-php-ext-enable imagick
+
+RUN apt-get install -y \
```
Comment (OskarStark) [inline 7.2/Dockerfile]: can you please do this here: https://github.com/exozet/docker-php-fpm/pull/31/files#diff-56373fa7c1a1d5caa179088d26e72d29R47

File: 7.2/Dockerfile [hunk 2]
```diff
@@ -84,3 +84,11 @@ WORKDIR /usr/src/app
...
+    ffmpeg \
+    supervisor
+
+RUN update-rc.d supervisor defaults
```
Comment (OskarStark) [inline 7.2/Dockerfile]: please use:
```
RUN update-rc.d supervisor defaults && \
    update-rc.d supervisor enable && \
    service supervisor restart
```
in this case only one layer is created by docker

**Comment by OskarStark** [comment]
@DracoBlue do we want to have both in the php fpm base images?
looks project specific to me, which should extend the base image. In this case you can create an automated build on docker hub an link the phpfpm image. Afterwards it will be build when the php base images changes... WDYT?

**Comment by niepi** [comment]
I agree with OskarStark, this should not be in the base image. If this should be public make a new image separate or tagged or create a private project specific image.

**Pull request closed without merging**